Makefile

```
@@ -20,7 +20,7 @@ install-base: install-base-pip-packages install-nltk-models
 install: install-base-pip-packages install-dev install-nltk-models install-test install-huggingface install-unstructured-inference
 
 .PHONY: install-ci
-install-ci: install-base-pip-packages install-test install-nltk-models install-huggingface install-unstructured-inference local-inference
+install-ci: install-base-pip-packages install-test install-nltk-models install-huggingface install-unstructured-inference install-local-inference
```

```suggestion
install-ci: install-base-pip-packages install-test install-nltk-models install-huggingface install-local-inference
```
Just realized we could remove `install-unstructured-inference` since it's a dep of `install-local-inference`.